doc/page-cmakefunctions.h

```diff
@@ -122,4 +122,17 @@ This function adds a test with the name \e TargetName.The parameter descriptions
 \param URI The URI of the Target i.e qml plugin
 \param VERSION The target i.e Qml plugin version, if not provided it defaults to 1.0
 
+\section cmake-params CMake options
+
+| Option                            | description                                                         |
+|-----------------------------------|---------------------------------------------------------------------|
+| FACELIFT_DISABLE_DBUS_IPC         | Disables IPC. The IPC part will not be built. Cannot use with FACELIFT_ENABLE_DBUS_IPC |
```

Comment:
This option disables the usage of DBUS. Some IPC classes are still available but no communication over DBUS is possible, which means no communication with another process is possible.

Reply:
So the statement is true then, there is no IPC. I will clarify more.